test/Client/Endpoint/IpEndpointTest.php

```
@@ -0,0 +1,53 @@
+            $this->mockClient,
+            new \GuzzleHttp\Psr7\HttpFactory(),
+            $this->streamFactory,
+        );
+    }
+
+    #[Test]
+    public function getIPFiltersBuildsCorrectRequest(): void
+    {
+        $this->mockJsonResponse(200, [['status' => 'ok']]);
+
+        $result = $this->endpoint->getIPFilters();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$result'. (undefined)

(UnusedLocalVariable)